src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/02-create-cluster-add-host.feature

```diff
@@ -0,0 +1,41 @@
+@login
+Feature: Cluster expansion host addition
+
+    Add some hosts and perform some host related actions like editing the labels
+    and removing the hosts from the cluster and verify all of the actions are performed
+    as expected
+
+    Background: Cluster expansion wizard
+        Given the welcome screen
+        Then go to 'wizard'
+        Then go to 'Add Host' section
+
+    Scenario Outline: Confirm there is an existing host, then add and remove hosts ceph-node-01 and ceph-node-02
+        Then verify host 'ceph-node-00' 'exists'
+        Then add host '<hostname>' with labels '<labels>'
+        Then remove host '<hostname>'
```

Comment:
The first one is a real outcome expectation (it'd be more clear as `Then a host 'ceph-node-00' exists`, but for reusability you can create a generic step that looks for rows in a table matching this: `Then a new (row|entry|item) appears in the (table|page) with values "ceph-node-00, IP, etc."`), but the other 2 are regular When steps (they just describe actions).